bucketers/uniform.py

```diff
@@ -30,6 +30,7 @@ def bucket(self, X):
                 bucket_size -= 1
             start, end = cur_start_index, cur_start_index + bucket_size
             end = X.shape[0] if end > X.shape[0] else end
+            print(X[start:end, :])
```

Comment:
Why are you adding this print?

Reply:
left it in by mistake.. will remove TODO